packages/create-turbo/src/commands/create/index.ts

```diff
@@ -92,6 +93,16 @@ export async function create(
     skipTransforms,
   });
 
+  const selectedPackageManager = selectedPackageManagerDetails.name;
+  if (selectedPackageManager !== "pnpm") {
```

Comment:
This shouldn't be a hardcoded check, sources aren't guaranteed to be `pnpm`, and in the case when the source isn't a first party example, I don't think we should run this transform at all. 

This should instead use the detected package manager, and only run with our own examples. We already have a transform that operates only on our own examples, so it could be added here! https://github.com/vercel/turbo/blob/main/packages/create-turbo/src/transforms/official-starter.ts

Reply:
the logic to update README has been added to the list of transformers that run when npm install (or any other equivalent command) is executed. Hence, no need for this check anymore.